app/controllers/api/v1/trends/statuses_controller.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
 class Api::V1::Trends::StatusesController < Api::BaseController
+  vary_by 'Authorization, Accept-Language'
+
   before_action :set_statuses
 
   after_action :insert_pagination_headers
 
   def index
+    cache_if_unauthenticated!
```

Comment:
This is very likely to cause inconsistent page cache, especially since trends pagination does not work with an underlying ID and depends on the rank of the trend items.